src/integrationTest/resources/envelope.json

```diff
@@ -20,7 +20,7 @@
           "file_name": "1111002.pdf",
           "notes": "test",
           "document_uuid": "0fa1ab60-f836-43aa-8c65-b07cc9bebcbe",
-          "document_type": "other"
+          "document_type": "form"
```

Comment:
Why this change?

Reply:
otherwise EnvelopeControllerTest is failing. @lgonczar

Reply:
test data looks incorrect to me we would never receive lower case `other` or `form` for document type.

Reply:
I don't see anything invalid or new about this. Why would it be failing?

Reply:
Didn't see Nitin's comment

Reply:
I've just checked and this is the expected envelope in DB, as returned by an endpoint. `other` is one of the valid values (see `DocumentType` enum). The expectation changed because input data is SSCS1. All good. I was confused by the file name.

Reply:
@lgonczar Exactly